### PR TITLE
fix: #911 using the undo shortcut after you clear the content goes into edit mode

### LINKED ISSUE #911: [Bug] 单元格设置格式（点击工具栏按钮）或使用右键菜单功能后，单元格失焦
### 初始清单

- [X] 这真的是个问题吗？
- [X] 我已经在 Github Issues 中搜索过了，但没有找到类似的问题。

### 受影响的包和版本

dev

### 复现步骤

case1、 选中单元格设置格式（点击工具栏按钮），然后输入内容
case2、切换子表后直接输入内容
case3、选中单元格击常规设置格式为货币，然后再输入内容
case4、右键菜单清除内容后，使用undo快捷键

### 预期行为
重新focus焦点

### 实际行为
不会重新focus焦点，需要选中后才能编辑
[2024_1_2 19_31_59 commands.json](https://github.com/dream-num/univer/files/13809471/2024_1_2.19_31_59.commands.json)
[2024_1_2 19_31_59 video.webm](https://github.com/dream-num/univer/assets/154207126/ab7b5837-4e79-4454-9db0-3dabf4516d9f)

### 运行环境

_No response_

### 操作系统

_No response_

### 构建工具

_No response_

### PR DESCRIPTION
**Pull request by Gggpound**
close #911

### CODE REVIEW

**Comment by Gggpound** [comment]
When you click on the toolbar or right-click on the menu, the editor loses focus. When the shortcut is triggered, the focus is on the body and execution is interrupted at the following location.

**Comment by Gggpound** [comment]
When you click on the toolbar or right-click on the menu, the editor loses focus. When the shortcut is triggered, the focus is on the body and execution is interrupted at the following location.

**Comment by Gggpound** [comment]
image

**Review by Jocs — APPROVED** [review]

File: packages/ui/src/views/components/context-menu/ContextMenu.tsx [hunk 1]
```diff
@@ -66,6 +68,8 @@ export function ContextMenu() {
                     onOptionSelect={(params) => {
                         const { label: commandId, value } = params;
                         commandService && commandService.executeCommand(commandId as string, { value });
+                        const textSelectionRenderManager = injector.get(ITextSelectionRenderManager);
+                        textSelectionRenderManager && textSelectionRenderManager.focus();
```
Comment (wzhudev) [inline packages/ui/src/views/components/context-menu/ContextMenu.tsx:72]: 没必要判空，如果 get 不到依赖注入系统已经报错了

**Pull request merged**